### DISCUSSION TITLE
--watch Not Rebuilding Upon Save

### DISCUSSION THREAD

**Discussion by muellerluke** [tailwindlabs/tailwindcss#14495] [Help]
**What version of Tailwind CSS are you using?**

v3.4.12

**What build tool (or framework if it abstracts the build tool) are you using?**

Golang http/template

**What version of Node.js are you using?**

v20.14.0

**What browser are you using?**

N/A

**What operating system are you using?**

Windows 11 Pro (Snapdragon X Elite ARM processor)

**Reproduction URL**

http://localhost:8080

**Describe your issue**

Upon changes to the /templates directory, tailwind does not rebuild. It's pretty simple.

**Comment by adamwathan** [comment]
Hey! Can you share a real reproduction? Obviously no one here can access a website running locally on your machine, and we need the code, not the site :)

**Reply by muellerluke** [reply]
@adamwathan after further investigation, it appears the issue is with any sort of file watching (if you want to call it that) with WSL on my current machine. I have a feeling this is likely a driver/compatibility issue with WSL for ARM processors.
